packages/components/src/index.js

```diff
@@ -8,6 +8,7 @@ export {
 	G,
 	HorizontalRule,
 	BlockQuotation,
+	PrimitiveButton,
```

Comment:
I left these exports here only for Backward compatibility, ideally we should just consume the primitives from the primitives package. 

Let's not expose other primitives here.